index.html

```diff
@@ -88,28 +88,28 @@ <h2>A project for first-time contributions</h2>
 
         <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
         <div class="card">
-          <p class="name">Your name</p>
+          <p class="name">Malcolm Wimbley</p>
           <p class="contact">
             <i class="fab fa-twitter"></i>
-            <a href="www.twitter.com" target="_blank">Your handle</a>
+            <a href="www.twitter.com/malcolm_wimbley" target="_blank">Malcolm_Wimbley</a>
           </p>
-          <p class="about">about you</p>
+          <p class="about">ReacJS genius, Call Of duty Top Sniper, Leetcode failer</p>
           <div class="resources">
             <p>3 Useful Dev Resources</p>
             <ul>
               <li>
-                <a href="#" target="_blank" title="First Resource">
-                  Resource 1
+                <a href="https://www.crackingthecodinginterview.com/contents.html" target="_blank" title="First Resource">
+                  Cracking the coding Interview
                 </a>
               </li>
               <li>
-                <a href="#" target="_blank" title="Second Resource">
-                  Resource 2
+                <a href="https://www.udemy.com/course/complete-react-developer-zero-to-mastery/" target="_blank" title="Second Resource">
+                  Udemy
                 </a>
               </li>
               <li>
-                <a href="#" target="_blank" title="Third resource">
-                  Resource 3
+                <a href="https://stackoverflow.com/" target="_blank" title="Third resource">
+                  Stack Over Flow
                 </a>
               </li>
             </ul>
```

Comment:
Copy the full template with the comments before and after the card and paste it after line 124 where it says: `DO NOT modify the TEMPLATE directly, make a copy and paste it below`